app-frontend/src/app/pages/admin/organization/users/users.module.js

```diff
@@ -128,6 +140,45 @@ class OrganizationUsersController {
             this.fetchUsers(1, this.search);
         });
     }
+
+    getUserGroupRole(user) {
```

Comment:
I would call this `getUserGroupRoleLabel` to make it clear that we're doing some logic to prepare it for display.